### PR TITLE
Revert "refs #299: Replace deprecated API used by Gradle plugin"

### PR DESCRIPTION
**Pull request by iloveeclipse**
This reverts commit 313ea5b65420e2c370ff5af6f82941622036d75e.

See #320 for details.

### CODE REVIEW

File: gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java [hunk 1]
```diff
@@ -193,7 +193,7 @@ protected void configureForSourceSet(final SourceSet sourceSet, SpotBugsTask tas
             public FileCollection call() {
                 // the simple "classes = sourceSet.output" may lead to non-existing resources directory
                 // being passed to SpotBugs Ant task, resulting in an error
-                return project.fileTree(sourceSet.getOutput().getClassesDirs()).builtBy(sourceSet.getOutput());
+                return project.fileTree(sourceSet.getOutput().getClassesDir()).builtBy(sourceSet.getOutput());
```
Comment (KengoTODA) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: MINOR Remove this use of "getClassesDir"; it is deprecated. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by KengoTODA** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java (hunk 1 above)
Comment (iloveeclipse) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: @KengoTODA : this is a **revert** commit, which fixes the problem with the new API.

Comment (KengoTODA) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: It's not me, posted by sonarqube.

Comment (KengoTODA) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: @hazendaz I will create a bot GitHub account for us, and update `GITHUB_TOKEN` for Travis.

Comment (KengoTODA) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: @iloveeclipse could you invite following user to this spotbugs organization? Then we can make message form SonarQube more intuitive.
https://github.com/spotbugs-bot

File: gradlePlugin/CHANGELOG.md [hunk 2]
```diff
@@ -7,7 +7,6 @@ This is the changelog for SpotBugs Gradle Plugin. This follows [Keep a Changelog
 ## 1.3 (2017/Aug/16)
 
 * Use SpotBugs 3.1.0-RC5
-* Stop using [single class directory](https://docs.gradle.org/4.0.2/release-notes.html#multiple-class-directories-for-a-single-source-set) to prepare for Gradle v5 ([#299](https://github.com/spotbugs/spotbugs/issues/299))
```
Comment (KengoTODA) [inline gradlePlugin/CHANGELOG.md]: we should not remove this line from CHANGELOG

File: gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java (hunk 1 above)
Comment (iloveeclipse) [inline gradlePlugin/src/main/java/com/github/spotbugs/SpotBugsPlugin.java]: I did this now.

File: gradlePlugin/CHANGELOG.md (hunk 2 above)
Comment (iloveeclipse) [inline gradlePlugin/CHANGELOG.md]: Done.

**Pull request merged**